app/views/application/_navbar.html.erb

```diff
@@ -9,13 +9,13 @@
     <div class="collapse navbar-collapse" id="cypressNavbar">
       <ul class="nav navbar-nav navbar-right">
         <% if user_signed_in? %>
-          <li <% if params[:controller] != 'records' && params[:controller] != 'registrations' %>class="active"<% end %>>
+          <li <% if (params[:controller] == 'records' && params[:product_test_id]) || (params[:controller] != 'records' && params[:controller] != 'registrations') %>class="active"<% end %>>
```

Comment:
http://stackoverflow.com/a/4354547 is a decent way to use the Rails routes to do just this, e.g. `current_page?(:some_route)`